cmd/disk-cache-backend.go

```diff
@@ -522,12 +598,45 @@ func (c *diskCache) bitrotReadFromCache(ctx context.Context, filePath string, of
 	return nil
 }
 
+// decryptCacheObjectETag tries to decrypt the ETag saved in encrypted format using the cache KMS
+func decryptCacheObjectETag(info *ObjectInfo) error {
```

Comment:
Can we define this method in `disk-cache-utils.go`? And also can we tag it as a method of `ObjectInfo`. Like,  
```
func (o *ObjectInfo) DecryptCacheObjectETag() error {...}
```

Reply:
moved this function to disk-cache-utils.go. There is no need to extend ObjectInfo struct with this method as this is usable only by cache and is not generic